packages/turbo-test-utils/src/validateLogs.ts

```diff
@@ -1,21 +1,13 @@
-import type { Spy } from "./spyConsole";
+import { expect } from "@jest/globals";
+import type { SpyConsole } from "./spyConsole";
+
+type Matcher = ReturnType<typeof expect.stringContaining>;
 
 export function validateLogs(
```

Comment:
so as you can see, I simplified `validateLogs` a lot.  for one thing, we weren't using it correctly (e.g. passing in just one value, and passing in a prefix even though we're also passing a matcher).  it came up because the types were very unhappy and I realized it can be done a lot more simply.

personally, I'd be a fan of just getting rid of it, it's only used in two files and I don't think it'd be terrible to just put the `toHaveBeenNthCalledWith` lines next to each other, but I wanted to keep the scope smaller so I left it for those cases where more than one line is passed in (because with just one, it doesn't make a lot of sense to use the helper).

Reply:
I'm fine with that, it's an ancient helper